README.md

```diff
@@ -1,14 +1,9 @@
-# Project
+# ONNXRuntime Swift Package Manager
```

Comment:
the name seems a bit odd to me. maybe "ONNX Runtime for Swift Package Manager"?

I feel similarly about the repo name, but I'm not sure how easy it is to change that. another idea for that is "onnxruntime-swift".

Reply:
I'm fine with 'ONNX Runtime' 

Not so fine with 'onnxruntime-swift' as this is specifically for consumption by the Swift Package Manage (vs. say a cocoapod with Swift bindings).